components/PlantForm.js

```diff
@@ -0,0 +1,181 @@
+import styled from "styled-components";
+import { uid } from "uid";
+
+export default function PlantForm({ onAddPlants, setDisplayForm }) {
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+    const randomId = uid();
+    const newPlant = {
+      id: `${randomId}`,
+      name: data.name,
+      botanicalName: data.botanicalName,
+      imageUrl:
+        "https://images.pexels.com/photos/2587313/pexels-photo-2587313.jpeg?auto=compress&cs=tinysrgb&w=1600",
+      waterNeed: data.waterNeed,
+      lightNeed: data.lightNeed,
+      fertiliserSeason: formData.getAll("fertiliserSeason"),
+      description: data.description,
+    };
+    onAddPlants(newPlant);
+    event.target.reset();
+    setDisplayForm(false);
+  }
+
+  return (
+    <>
+      <p>Create</p>
+      <StyledForm onSubmit={handleSubmit}>
+        <label htmlFor="input-name">Plant Name</label>
+        <input
+          id="input-name"
+          type="text"
+          name="name"
+          aria-label="Input for Plant Name"
+          required
+        />
+
+        <label htmlFor="input-botanical-name">Botanical Name</label>
+        <input
+          id="input-botanical-name"
+          type="text"
+          name="botanicalName"
+          aria-label="Input for Plant Botanical Name"
+          required
+        />
+
+        <label htmlFor="description">Plant Description</label>
+        <textarea
+          id="description"
+          name="description"
+          aria-label="Textinput for Plant Description"
+          placeholder="Tell us something about your plant"
+        />
+
+        <fieldset>
+          <legend>Light Needs</legend>
+          <label htmlFor="full-shade">
```

Comment:
```suggestion
          <label>
```
When you wrap the label around the input you don't need to connect it via 'htmlFor'.